src/enums.rs

```diff
@@ -47,6 +47,7 @@ pub enum Instrument {
     M20HeliRte,
     NsytICC,
     NsytIDC,
+    #[default]
```

Comment:
This is good, I didn't know about `Default` for enums. (perhaps that's what you're doing below with the empty initialization?)   I might get rid of `None` as an enum value and replace with `Option<Instrument>`

Reply:
I just added it because it was simple enough and is nice to have.